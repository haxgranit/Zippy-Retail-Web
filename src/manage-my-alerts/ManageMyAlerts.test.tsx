import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { Modal } from 'react-bootstrap';
import ManageMyAlerts from './ManageMyAlerts';
import EditBusinessPhone from './edit-business-phone/EditBusinessPhone';
import EditHomePhone from './edit-home-phone/EditHomePhone';

// Configure enzyme for react 17
Enzyme.configure({ adapter: new Adapter() });

describe('Manage Alert Component', () => {
  it('Click Edit Home Phone button  ', () => {
    const wrapper = shallow(<ManageMyAlerts />);
    expect(wrapper.find(EditHomePhone).prop('show')).toBe(false);
    wrapper.find('Button[variant="link"]').at(2).simulate('click');
    expect(wrapper.find(EditHomePhone).prop('show')).toBe(true);
  });
  it('Click Edit Business Phone  button ', () => {
    const wrapper = shallow(<ManageMyAlerts />);
    expect(wrapper.find(EditBusinessPhone).prop('show')).toBe(false);
    wrapper.find('Button[variant="link"]').at(3).simulate('click');
    expect(wrapper.find(EditBusinessPhone).prop('show')).toBe(true);
  });
});
