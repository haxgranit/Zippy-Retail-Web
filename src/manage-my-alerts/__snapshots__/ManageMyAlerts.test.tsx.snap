// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manage Alert Component Render manage-my-alerts page 1`] = `
<div>
  <div
    class="d-flex justify-content-between align-items-center"
  >
    <h3>
      MANAGE MY ALERTS
    </h3>
    <div
      class="d-flex"
    >
      <div
        style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
        title="print"
      >
        P
      </div>
      <div
        style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
        title="copy"
      >
        P
      </div>
      <div
        style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
        title="question"
      >
        P
      </div>
    </div>
  </div>
  <div
    class="mt-3 row"
  >
    <div
      class="col-md-8"
    >
      <ul>
        <li>
          Alerts are notifictions that ZippyX sends to keep you informed of transactions and monitor your ZippyX accounts.
        </li>
        <li>
          Most alerts are sent automatically to My Messages by default and to any other additional contact methods you select and save.
        </li>
        <li>
          If you choose to activate Push Notifications, you must also enable them on your mobile device.
        </li>
        <li>
          Push Notifications are personalized alerts sent directly to your mobile device that can help you stay on top of your ZippyX accounts. If you sign into ZippyX Mobile Banking from a mobile device other than your own, Your alerts may be sent to that device until another user signs in to the application.
        </li>
        <li>
          Some fraud prevention alerts cannot be turned off.
        </li>
      </ul>
      <ul
        class="mt-2 mb-3 nav nav-tabs"
        role="tablist"
      >
        <li
          class="nav-item"
          role="presentation"
        >
          <button
            aria-controls="status-tab-tabpane-fraud"
            aria-selected="true"
            class="nav-link active"
            data-rr-ui-event-key="fraud"
            id="status-tab-tab-fraud"
            role="tab"
            type="button"
          >
            FRAUD PREVENTION
          </button>
        </li>
        <li
          class="nav-item"
          role="presentation"
        >
          <button
            aria-controls="status-tab-tabpane-transactions"
            class="nav-link"
            data-rr-ui-event-key="transactions"
            id="status-tab-tab-transactions"
            role="tab"
            tabindex="-1"
            type="button"
          >
            TRANSACTIONS
          </button>
        </li>
        <li
          class="nav-item"
          role="presentation"
        >
          <button
            aria-controls="status-tab-tabpane-reminders"
            class="nav-link"
            data-rr-ui-event-key="reminders"
            id="status-tab-tab-reminders"
            role="tab"
            tabindex="-1"
            type="button"
          >
            REMINDERS
          </button>
        </li>
      </ul>
      <div
        class="tab-content"
      >
        <div
          aria-labelledby="status-tab-tab-fraud"
          class="tab-pane active"
          id="status-tab-tabpane-fraud"
        >
          <div
            class="row"
            style="background-color: rgb(239, 239, 239); border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <p
                style="margin-top: 12px;"
              >
                GET ALERTED WHEN...
              </p>
            </div>
            <div
              class="col-md-4"
            >
              <p
                style="margin-top: 12px;"
              >
                SEND ALERT TO...
              </p>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                There was unusual activity on my card
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  My Messages
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Email
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Text Message
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Push Notification
                </label>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                Changes have been made to my Online, Mobile and Telephone Banking passwords
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  My Messages
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Email
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Text Message
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Push Notification
                </label>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                My personal information has been updated on my profile
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  My Messages
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Email
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Text Message
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Push Notification
                </label>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                A bill payee or Interac e-Transfer contact was added
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
            <div
              class="col-md-4"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  My Messages
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Email
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Text Message
                </label>
              </div>
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Push Notification
                </label>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-12"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  There waas an attempt to exceed my ATM daily
                </label>
              </div>
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                withdrawal limit
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="d-flex col-md-12"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  There was an attempt to exceed your Point of Sale daily limit
                </label>
              </div>
              <div
                style="font-weight: bold;"
              >
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-12"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A bank machine withdrawal is more than this amount:
                </label>
              </div>
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                <div
                  class="mb-3 input-group"
                  style="width: 200px;"
                >
                  <span
                    class="input-group-text"
                    id="basic-addon1"
                  >
                    $
                  </span>
                  <input
                    aria-describedby="basic-addon1"
                    class="form-control"
                  />
                </div>
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
          </div>
          <div
            class="row"
            style="padding: 10px; border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-12"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  There was a debit card purchase more than:
                </label>
              </div>
              <div
                class="d-flex"
                style="font-weight: bold;"
              >
                <div
                  class="mb-3 input-group"
                  style="width: 200px;"
                >
                  <span
                    class="input-group-text"
                    id="basic-addon1"
                  >
                    $
                  </span>
                  <input
                    aria-describedby="basic-addon1"
                    class="form-control"
                  />
                </div>
                <div
                  style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                >
                  P
                </div>
              </div>
            </div>
          </div>
          <div
            class="mt-4 p-3"
            style="border-top: 1px solid #e5e5e5; border-bottom: 1px solid #e5e5e5;"
          >
            <div
              class="d-flex justify-content-end row"
            >
              <button
                class="d-flex btn btn-outline-danger"
                style="width: auto; margin-right: 10px;"
                type="button"
              >
                Reset
              </button>
              <button
                class="d-flex btn btn-danger"
                style="width: auto; margin-right: 10px;"
                type="button"
              >
                Save
              </button>
            </div>
          </div>
          <div
            class="mt-2"
          >
            <div
              class="d-flex"
            >
              <div
                style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
              >
                P
              </div>
              <span
                style="font-weight: bold;"
              >
                Note:
              </span>
            </div>
            <div
              class="d-flex flex-column"
            >
              <ul>
                <li>
                  An inactive alert status applies to bank accoiunts that were automatically transferred from Paperless to eStatements. If your alert status is displayed as 'Inactive', it means you have not agreed to the terms and conditions for eStatements. Once you have accepted the terms and conditions, you alerts will be activated and you will not see an 'Inactive' alert status.
                </li>
                <li>
                  In the case of credit cards, your alerts status will always be active.
                </li>
                <li>
                  ZippyX Alerts Terms and Conditions are now a part of the
                  <a
                    class="text-black"
                    href="/"
                  >
                    Electronic Access Agreement
                  </a>
                </li>
              </ul>
              <div
                class="row"
              >
                <div
                  class="d-flex align-items-center mt-2 col-md-12"
                >
                  <div
                    style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                  >
                    P
                  </div>
                  <span>
                    eStatement Terms and Conditions (last updated: November 2011) (PDF, 145KB).
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-labelledby="status-tab-tab-transactions"
          class="tab-pane"
          id="status-tab-tabpane-transactions"
        >
          <div
            class="row"
            style="background-color: rgb(239, 239, 239); border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <p
                style="margin-top: 12px;"
              >
                GET ALERTED WHEN...
              </p>
            </div>
            <div
              class="col-md-4"
            >
              <p
                style="margin-top: 12px;"
              >
                SEND ALERT TO...
              </p>
            </div>
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A scheduled bill payment has failed
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A scheduled bill payment was successful
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A scheduled transfer has failed
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A scheduled transfer was successful
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  An interac e-Transfer was sent
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A return has been made
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  Held funds are now available
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            class="d-flex justify-content-end row"
          >
            <button
              class="btn btn-outline-danger"
              style="width: auto; margin-right: 10px;"
              type="button"
            >
              Reset
            </button>
            <button
              class="btn btn-danger"
              style="width: auto; margin-right: 10px;"
              type="button"
            >
              Save
            </button>
          </div>
          <div
            class="mt-2"
          >
            <hr
              style="margin-top: 3rem; height: 2px;"
            />
            <div
              class="d-flex"
            >
              <div
                style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
              >
                P
              </div>
              <span
                style="font-weight: bold;"
              >
                Note:
              </span>
            </div>
            <div
              class="d-flex flex-column"
            >
              <ul>
                <li>
                  An inactive alert status applies to bank accoiunts that were automatically transferred from Paperless to eStatements. If your alert status is displayed as 'Inactive', it means you have not agreed to the terms and conditions for eStatements. Once you have accepted the terms and conditions, you alerts will be activated and you will not see an 'Inactive' alert status.
                </li>
                <li>
                  In the case of credit cards, your alerts status will always be active.
                </li>
                <li>
                  ZippyX Alerts Terms and Conditions are now a part of the
                  <a
                    class="text-black"
                    href="/"
                  >
                    Electronic Access Agreement
                  </a>
                </li>
              </ul>
              <div
                class="row"
              >
                <div
                  class="d-flex align-items-center mt-2 col-md-12"
                >
                  <div
                    style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                  >
                    P
                  </div>
                  <span>
                    eStatement Terms and Conditions (last updated: November 2011) (PDF, 145KB).
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          aria-labelledby="status-tab-tab-reminders"
          class="tab-pane"
          id="status-tab-tabpane-reminders"
        >
          <div
            class="row"
            style="background-color: rgb(239, 239, 239); border-bottom: 1px solid grey;"
          >
            <div
              class="col-md-8"
            >
              <p
                style="margin-top: 12px;"
              >
                GET ALERTED WHEN...
              </p>
            </div>
            <div
              class="col-md-4"
            >
              <p
                style="margin-top: 12px;"
              >
                SEND ALERT TO...
              </p>
            </div>
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A postdated bill advance notification was sent
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A reminder that an upcoming scheduled transfer is due
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A new message is available in My Messages
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  A scheduled transfer was successful
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <div
            style="margin: 20px 5px;"
          >
            <div
              class="d-flex"
            >
              <div
                class="form-check"
              >
                <input
                  class="form-check-input"
                  type="checkbox"
                />
                <label
                  class="form-check-label"
                  title=""
                >
                  There are new documents available in your ZippyX Dgital Vault
                </label>
              </div>
              <i />
            </div>
            <hr
              style="height: 2px;"
            />
          </div>
          <span
            style="font-size: 20px;"
          >
            <span
              style="color: rgb(239, 53, 69); margin-right: 5px;"
            >
              +
            </span>
            Notice of change
          </span>
          <hr
            style="height: 2px;"
          />
          <div
            class="d-flex justify-content-end row"
          >
            <button
              class="btn btn-outline-danger"
              style="width: auto; margin-right: 10px;"
              type="button"
            >
              Reset
            </button>
            <button
              class="btn btn-danger"
              style="width: auto; margin-right: 10px;"
              type="button"
            >
              Save
            </button>
          </div>
          <div
            class="mt-2"
          >
            <hr
              style="margin-top: 3rem; height: 2px;"
            />
            <div
              class="d-flex"
            >
              <div
                style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
              >
                P
              </div>
              <span
                style="font-weight: bold;"
              >
                Note:
              </span>
            </div>
            <div
              class="d-flex flex-column"
            >
              <ul>
                <li>
                  An inactive alert status applies to bank accoiunts that were automatically transferred from Paperless to eStatements. If your alert status is displayed as 'Inactive', it means you have not agreed to the terms and conditions for eStatements. Once you have accepted the terms and conditions, you alerts will be activated and you will not see an 'Inactive' alert status.
                </li>
                <li>
                  In the case of credit cards, your alerts status will always be active.
                </li>
                <li>
                  ZippyX Alerts Terms and Conditions are now a part of the
                  <a
                    class="text-black"
                    href="/"
                  >
                    Electronic Access Agreement
                  </a>
                </li>
              </ul>
              <div
                class="row"
              >
                <div
                  class="d-flex align-items-center mt-2 col-md-12"
                >
                  <div
                    style="width: 20px; height: 25px; border: 1px dotted grey; text-align: center; margin-right: 10px;"
                  >
                    P
                  </div>
                  <span>
                    eStatement Terms and Conditions (last updated: November 2011) (PDF, 145KB).
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="col-md-4"
    >
      <div
        class="bg-white card"
      >
        <div
          class="card-body"
        >
          <h5
            class="mt-4 mb-3"
          >
            Contact Information
          </h5>
          <div
            class="d-flex justify-content-between"
            style="border-bottom: 1px solid grey; padding-top: 5px; padding-bottom: 5px;"
          >
            <div>
              <div
                style="font-weight: bold;"
              >
                Email
              </div>
              <div
                style="color: grey;"
              >
                dtough@hotmail.com
              </div>
            </div>
            <button
              class="text-black btn btn-link"
              type="button"
            >
              Edit
            </button>
          </div>
          <div
            class="d-flex justify-content-between"
            style="border-bottom: 1px solid grey; padding-top: 5px; padding-bottom: 5px;"
          >
            <div>
              <div
                style="font-weight: bold;"
              >
                SMS mobile phone
              </div>
              <div
                style="color: grey;"
              >
                410-553-1372
              </div>
            </div>
            <button
              class="text-black btn btn-link"
              type="button"
            >
              Edit
            </button>
          </div>
          <div
            class="d-flex justify-content-between"
            style="border-bottom: 1px solid grey; padding-top: 5px; padding-bottom: 5px;"
          >
            <div>
              <div
                style="font-weight: bold;"
              >
                Home Phone
              </div>
              <div
                style="color: grey;"
              >
                410-553-1372
              </div>
            </div>
            <button
              class="text-black btn btn-link"
              type="button"
            >
              Edit
            </button>
          </div>
          <div
            class="d-flex justify-content-between"
            style="padding-top: 5px; padding-bottom: 5px;"
          >
            <div>
              <div
                style="font-weight: bold;"
              >
                Business Phone
              </div>
            </div>
            <button
              class="text-black btn btn-link"
              type="button"
            >
              Edit
            </button>
          </div>
        </div>
      </div>
      <div
        class="bg-white mt-4 card"
      >
        <div
          class="card-body"
        >
          <h5>
            Did you know?
          </h5>
          <p>
            You can avoid late credit card payments when you receive your credit card minimum payment amount and due date via email or SMS
          </p>
          <a
            class="text-decoration-none text-black"
            href="/"
            style="margin-top: 10px; margin-bottom: 5px; display: block; font-size: 15px;"
          >
            Turns on Minimum Payment Due alert now
            <span
              style="color: rgb(220, 53, 69); margin-left: 5px;"
            >
              &gt;
            </span>
          </a>
          <a
            class="text-decoration-none text-black"
            href="/"
            style="margin-top: 10px; margin-bottom: 5px; display: block; font-size: 15px;"
          >
            Apply for a ZippyX Credit Card today
            <span
              style="color: rgb(220, 53, 69); margin-left: 5px;"
            >
              &gt;
            </span>
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
